test/Microsoft.Extensions.Configuration.Binder.Test/ConfigurationBinderTests.cs

```diff
@@ -73,6 +74,40 @@ public class GenericOptions<T>
             public T Value { get; set; }
         }
 
+        public class ConfigurationInterfaceOptions
+        {
+            public IConfigurationSection Section { get; set; }
+        }                
+
+        [Fact]
+        public void CanBindIConfigurationSection()
+        {
+            var dic = new Dictionary<string, string>
+            {
+                {"Section:Integer", "-2"},
+                {"Section:Boolean", "TRUe"},
+                {"Section:Nested:Integer", "11"},
+                {"Section:Virtual", "Sup"}
+            };
+            var configurationBuilder = new ConfigurationBuilder();
+            configurationBuilder.AddInMemoryCollection(dic);
+            var config = configurationBuilder.Build();
+
+            var options = config.Get<ConfigurationInterfaceOptions>();
+
+            var childOptions = options.Section.Get<DerivedOptions>();
+
+            Assert.True(childOptions.Boolean);
+            Assert.Equal(-2, childOptions.Integer);
+            Assert.Equal(11, childOptions.Nested.Integer);
+            Assert.Equal("Derived:Sup", childOptions.Virtual);
+
+            Assert.Equal("Section", options.Section.Key);
+            Assert.Equal("Section", options.Section.Path);
+            Assert.Equal(null, options.Section.Value);
+        }
+
+
```

Comment:
Can you add one more test where the DerviedOptions that you bind from the first toplevel section, itself has another IConfigurationSection inside of it?